utils/query_helpers.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+from langchain.callbacks import get_openai_callback
+from yolopandas import pd
+
+
+def run_query_with_cost(df: pd.DataFrame, query: str, yolo: bool = False):
```

Comment:
Missing return type (I think you imported `Any` above for this)

Reply:
Added thanks.